pom.xml

```diff
@@ -0,0 +1,373 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.bahmni.module</groupId>
+	<artifactId>bahmni-events</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>Bahmni Events</name>
+	<description>Bahmni events module to send business events which can be used by external services</description>
+	<url>https://demo.mybahmni.org/bahmni/home/index.html</url>
+
+	<licenses>
+		<license>
+			<name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
+			<url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
+		</license>
+	</licenses>
+	<developers>
+		<developer>
+			<name>Bahmnitw</name>
+			<email>bahmnicommunity@gmail.com</email>
+			<organization>Bahmni</organization>
+			<organizationUrl>https://github.com/Bahmni</organizationUrl>
+		</developer>
+	</developers>
+	<scm>
+		<connection>scm:git:git@github.com:riteshghiya90/bahmni-events.git</connection>
+		<developerConnection>scm:git:https://github.com/riteshghiya90/bahmni-events.git</developerConnection>
+		<url>https://github.com/riteshghiya90/bahmni-events/</url>
+	</scm>
+
+	<modules>
+		<module>api</module>
+		<module>omod</module>
+	</modules>
+
+	<properties>
+		<openmrsPlatformVersion>2.5.10</openmrsPlatformVersion>
```

Comment:
If all versions are already specified here, then maybe other pom.xml should just reference this using { } and not specify them again.

Reply:
yes done the changes